gcp_deepvariant_runner_test.py

```diff
@@ -211,12 +211,15 @@ def testRunPipeline_WithGVCFOutFile(self, mock_can_write_to_bucket,
   @mock.patch.object(multiprocessing, 'Pool')
   @mock.patch('gcp_deepvariant_runner._gcs_object_exist')
   @mock.patch('gcp_deepvariant_runner._can_write_to_bucket')
-  def testRunMakeExamples(self, mock_can_write_to_bucket, mock_obj_exist,
-                          mock_pool):
+  @mock.patch('tempfile.NamedTemporaryFile')
+  def testRunMakeExamples(self, mock_temp_file, mock_can_write_to_bucket,
+                          mock_obj_exist, mock_pool):
     mock_apply_async = mock_pool.return_value.apply_async
     mock_apply_async.return_value = None
     mock_obj_exist.return_value = True
     mock_can_write_to_bucket.return_value = True
+    # TODO(samanvp): it will be a stronger test if we validate content of file.
```

Comment:
TODOs should point to an issue.

eg

TODO(#1234): Update test to validate the content of the file.

And in that issue, explain why it is too hard to just do now (or just do it now :)

Reply:
I removed the TODOs and added the json files verification to the unit tests.